io.catenax.shared.business_partner_certificate/1.0.0/BusinessPartnerCertificate.ttl

```diff
@@ -0,0 +1,171 @@
+#######################################################################
+# Copyright (c) 2023-2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.business_partner_certificate:1.0.0#> .
+@prefix ext-number: <urn:samm:io.catenax.shared.business_partner_number:2.0.0#> .
+
+:BusinessPartnerCertificate a samm:Aspect ;
+   samm:preferredName "BusinessPartnerCertificate"@en ;
+   samm:description "A business partner certifcate describes a certification (eg ISO9001, IATF-16949) via a certifcate document  of a Catena-X business partner"@en ;
+   samm:properties ( :businessPartnerNumber :type :registrationNumber [samm:property :areaOfApplication; samm:optional true] [samm:property :enclosedSites; samm:optional true] :validFrom :validUntil [samm:property :issuer; samm:optional true] :trustLevel [samm:property :validator; samm:optional true] [samm:property :uploader; samm:optional true] :documentID) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+ 
+:businessPartnerNumber a samm:Property ;
+   samm:preferredName "BusinessPartnerNumber"@en ;
+   samm:description "The BPN of the certified legal entity "@en ;
+   samm:characteristic ext-number:BpnlTrait ;
+   samm:exampleValue "BPNL00000003AYRE".
+
+
+ :type a samm:Property;
+    samm:preferredName "Type"@en;
+    samm:description "Type of the certificatation as defined on the document like IS09001, IATF 16949 or other"@en;
+    samm:characteristic [ a samm-c:SingleEntity ; samm:dataType :certificateTypeEntity].
+	
+	
+ :certificateTypeEntity a samm:Entity;

```

Comment:
the identifiers for all model elements start with a capital letter except for properties. So `certificateTypeEntity` and other identifiers should start with capital letter.